service/controller/v14/resource/cloudformation/current.go

```diff
@@ -67,7 +67,17 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 		var hostedZoneNameServers string
 		if r.route53Enabled {
 			hostedZoneNameServers, err = sc.CloudFormation.GetOutputValue(stackOutputs, key.HostedZoneNameServers)
-			if err != nil {
+			// TODO This is for migration from v13 to v14. Remove this check when all guest clusters are managed by v14 or newer.
```

Comment:
Can we track this with an issue please?

Reply:
I added point here https://github.com/giantswarm/giantswarm/issues/3314

```
Fix v14/resource/cloudformation/current.go TODO
````